content/applications/sales/crm/pipeline/lost_opportunities.rst

```diff
@@ -45,6 +45,12 @@ corner of the opportunity.
    :align: center
    :alt: A lost opportunity with the lost banner added.
 
+.. note::
+   Only *active* opportunities can be marked as *Won* or *Lost*.
+
+   To mark an *inactive* opportunity as lost, first set the probability of the opportunity to 0%.
+   The :guilabel:`Lost` button can now be used to mark the opportunity as lost.
+
```

Comment:
>    Only *active* opportunities can be marked as *Won* or *Lost*.

Just confirmed in runbot— I don't think this is true, as I was able to mark an inactive lead as Lost.

>    The :guilabel:`Lost` button can now be used to mark the opportunity as lost.

The **Lost** button will only appear again if the opp is unarchived. Setting the prob % to 0 automatically marks it as lost.

```suggestion
   To mark an *inactive* (archived) opportunity as lost, set the :guilabel:`Probability` field to `0` percent.
```